op-guide/horizontal-scale.md

```diff
@@ -0,0 +1,105 @@
+# Scaling a TiDB cluster
+
+## Overview
+
+The capacity of a TiDB cluster can be incresead or reduced without affecting online services.
+
+The following part shows you how to increase or delete PD, TiKV or TiDB nodes.
+
+About pd-ctl document, please refer to [pd-control](./pd-control.md).
+
+## PD
+
+Assume we have three PD servers with the following details:
+
+|Name|ClientUrls|PeerUrls|
+|----|----------|--------|
+|pd1|http://host1:2379|http://host1:2380|
+|pd2|http://host2:2379|http://host2:2380|
+|pd3|http://host3:2379|http://host3:2380|
+
+Get the information about the existing PD nodes through pd-ctl:
+
+```bash
+./pd-ctl -u http://host1:2379
+>> member
+```
+
+### Add a node dynamically
+
+Add a new PD server to the current PD custer by using the parameter `join`.
+To add `pd4`, you just need to specify the client url of any PD server in the PD cluster in the parameter `--join`, like:
+
+```bash
+./bin/pd-server --name=pd4 \
+                --client-urls="http://host4:2379" \
+                --peer-urls="http://host4:2380" \
+                --join="http://host1:2379"
+```
+
+### Delete a node dynamically
+
+Delete `pd4` through pd-ctl:
+
+```bash
+./pd-ctl -u http://host1:2379
+>> member delete pd4
+```
+
+### Migrate a node dynamically
+To migrate all PD nodes to a new machine, you need to add nodes on the new machine and then delete nodes on the old machine.
+In the process of migration, you should migrate the nodes one by one. After completing one step, you can verify the process by checking the information of all nodes.
```

Comment:
I think it will be more clear to use singular form in the first paragraph.

> To migrate a PD node to a new machine, you need to add a node...
> In the process of migrating multiple nodes, ...